developer_guides/linux_driver/third_party/keyword_detect/keyword_detect.rst

```diff
@@ -0,0 +1,140 @@
+.. _sof_driver_arch:
+
+Keyword Detection Driver implementation and user guide
+######################################################
+
+.. contents::
```

Comment:
Can you add a small introduction of what keyword detection is and how it is supposed to work? Also it is not very clear how the detection algorithm is loaded and used by SOF.

Reply:
Sure, done now.